odinmain/src/main/kotlin/me/odinmain/features/impl/dungeon/BloodCamp.kt

```diff
@@ -205,11 +206,17 @@ object BloodCamp : Module(
                 currVector.zCoord + speedVectors.zCoord * ping
             )
 
```

Comment:
cant you use your new function for the endPoint Vec3?

Reply:
the new function needs the endpoint vec, but i may be misunderstanding

Reply:
like use the function you added to calculate it, it looks like the exact same calculation